src/pages/AccessResultRecords.js

```diff
@@ -624,25 +615,21 @@ class AccessResultRecords extends Component {
       this.electionRPCall(electionId, false) //Election.findRPElectionReview(electionId, false (finalStatus))
     ]);
 
+    const datasets = await DataSet.getDarDatasets(darAPIResults.darInfo.datasetIds);
     const {darData, electionReview, showDULData, vaultVote} = electionReviewResults;
-    const restrictions = darAPIResults.darInfo.restrictions;
     const consent = electionReview.consent;
 
     assignToState = Object.assign({}, darData, electionReview, showDULData, vaultVote, electionAPIResults, darAPIResults, finalDACVote, electionRPResults);
-    assignToState.hasUseRestrictions = !isNil(restrictions) && !isEmpty(restrictions);
     assignToState.consentName = consent.name;
     assignToState.dataUse = consent.dataUse;
-    //NOTE: need to add hasUseRestriction to the state object, but is there a way for me to pull it forward?
+    assignToState.datasets = datasets;
+    assignToState.researcherProfile = darAPIResults.researcherProfile;
 
     //NOTE: lot of election calls, they seem to have different contexts and have extra stuff added to them
     //On top of that it just gets a bunch of other stuff (votes, consent) objects attached to it and returned as a seperate data structure
     //Contextually the difference between each of these elections are not clear (TYPE is different, but what that entails is not obvious)
     //Similar questions with the various votes (rpVotes, finalDACVote).
 
-    //NOTE: the component currently uses hasRestrictions, which I found out to simply be a check on the restrictions attribute on a DAR
-    //Basically all it does is check if a value exists and returns the bool value
-    //So all I've done is simply check that value myself from the describeDar return value rather than use the endpoint
-
```

Comment:
The original code was incorrectly testing if a restriction existed. A restriction **always** exists.